build.gradle

```diff
@@ -133,12 +133,13 @@ dependencies {
     // TODO: has an internal dependency exporting org.w3c.dom, temporarily removed
     //compile 'de.codecentric.centerdevice:javafxsvg:1.3.0'
     compile 'de.jensd:fontawesomefx-materialdesignfont:1.7.22-4'
+    compile 'de.jensd:fontawesomefx-commons:9.1.2'
     compile 'de.saxsys:mvvmfx-validation:1.7.0'
     compile 'de.saxsys:mvvmfx:1.7.0'
     compile 'org.fxmisc.easybind:easybind:1.0.3'
     compile 'org.fxmisc.flowless:flowless:0.6.1'
     compile 'org.fxmisc.richtext:richtextfx:0.9.0'
-    compile 'com.sibvisions.external.jvxfx:dndtabpane:0.1'
+//    compile 'com.sibvisions.external.jvxfx:dndtabpane:0.1'
```

Comment:
Dnd TabPane has been introduced offically with java 10:
https://bugs.openjdk.java.net/browse/JDK-8190717

Reply:
Good to know! Java 9 removed some (internal) classes this library uses. 
I don't think we'll be able to have the D'n'd feature in a Java 9 release unless someone patches the library.  But as Java 9 is already superseded, I guess this is not really high priority.

Reply:
Agree with you. As soon we have it running with java 9, we can focus on the migration to java 10 ;)